cmd/kube-controller-manager/app/autoscaling.go

```diff
@@ -40,7 +40,7 @@ func startHPAController(ctx ControllerContext) (bool, error) {
 	)
 	replicaCalc := podautoscaler.NewReplicaCalculator(metricsClient, hpaClient.Core())
 	go podautoscaler.NewHorizontalController(
-		hpaClient.Core(),
+		ctx.ClientBuilder.ClientGoClientOrDie("horizontal-pod-autoscaler").Core(),
```

Comment:
> this belongs to the PR?

sadly yes.  the client types aren't compatible.  Need to be able to get each type of client until we fully shift.